doc/proposals/ansible-operator-status.md

```diff
@@ -1,9 +1,20 @@
+
 # Ansible Operator Status Proposal
 
 > Status: **completed**
```

Comment:
To be consistent, we should decide to either capitalize status values or not across the board, and use present tense or past tense consistently.
```suggestion
> Status: **Complete**
```